src/svg-loader-components/audio.js

```diff
@@ -1,7 +1,9 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 
-const Audio = ({ className, ...props }) => (
+const Audio = ({
+  className = undefined, // String
```

Comment:
I added the `// String` comment to these in case you decide to implement type checking using TypeScript, Flow or some other solution.